pkg/daemon/ceph/client/subvolumegroup.go

```diff
@@ -28,10 +28,19 @@ import (
 
 // CreateCephFSSubVolumeGroup create a CephFS subvolume group.
 // volName is the name of the Ceph FS volume, the same as the CephFilesystem CR name.
-func CreateCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName, groupName string) error {
+func CreateCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName, groupName string, svgSpec *cephv1.CephFilesystemSubVolumeGroupSpec) error {
 	logger.Infof("creating cephfs %q subvolume group %q", volName, groupName)
-	//  [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]
+	// [<size:int>] [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]
 	args := []string{"fs", "subvolumegroup", "create", volName, groupName}
```

Comment:
So you've tested calling create on an existing volume when just updating the size and the size was updated? The [svg doc](https://docs.ceph.com/en/latest/cephfs/fs-volumes/) shows a command for resizing the volume so I assumed calling `create` again would not update the size.

Reply:
```
sh-4.4$ ceph fs subvolumegroup create myfs rook-test --size=1024
sh-4.4$ ceph fs subvolumegroup info myfs rook-test
{
    "atime": "2024-04-10 10:20:23",
    "bytes_pcent": "0.00",
    "bytes_quota": 1024,
    "bytes_used": 0,
    "created_at": "2024-04-10 10:20:23",
    "ctime": "2024-04-10 10:20:23",
    "data_pool": "myfs-replicated",
    "gid": 0,
    "mode": 16877,
    "mon_addrs": [
        "10.98.44.171:6789"
    ],
    "mtime": "2024-04-10 10:20:23",
    "uid": 0
}
sh-4.4$ ceph fs subvolumegroup create myfs rook-test --size=2046
sh-4.4$ ceph fs subvolumegroup info myfs rook-test
{
    "atime": "2024-04-10 10:20:23",
    "bytes_pcent": "0.00",
    "bytes_quota": 2046,
    "bytes_used": 0,
    "created_at": "2024-04-10 10:20:23",
    "ctime": "2024-04-10 10:20:37",
    "data_pool": "myfs-replicated",
    "gid": 0,
    "mode": 16877,
    "mon_addrs": [
        "10.98.44.171:6789"
    ],
    "mtime": "2024-04-10 10:20:23",
    "uid": 0
}
```

Create does update the size as well.

Reply:
create even shrink the size which is not good, updated code to use resize which avoid shrinking problem